webpack.config.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path');
+const TerserPlugin = require('terser-webpack-plugin');
+
+const masterVersion = require('./package.json').version;
+const packagesDir = path.resolve(__dirname, 'packages');
+const packageDir = path.resolve(packagesDir, 'vueposu');
```

Comment:
reanme to `vueposuDir`

Reply:
or just `path.resolve(__dirname, 'packages', 'vueposu')`

Reply:
it's too similar of this two variables

Reply:
Now just do some test in here, it's not sure yet.